README.md

```diff
@@ -65,138 +30,36 @@ After you're finished please remove all the comments and instructions!
 
 > The very first project of the 1st module of microverse full stack web development program
 
-**[your_project__name]** is a...
-
-## 🛠 Built With <a name="built-with"></a>
+**[first-microverse-project]** is a simple HTML project
 
-### Tech Stack <a name="tech-stack"></a>
+## 🛠 Built With <a name="built-with">HTML</a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+### Tech Stack <a name="tech-stack">CSS</a>
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
+- **[To get in touch with Git Hub]**
+- **[To be more familiar with Git Bash]**
+- **[Understand how HTML and CSS work together]**
```

Comment:
- [ ] Again, as my colleague opined, kindly consider removing the braces around your texts except it is a link (which will be immediately followed by a bracket) unlike this case. This allows readers to understand you and read your documentation with ease.